### PR TITLE
feat: add signup page [SV-102]

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
### 🔗 Linked issue

https://vsf.atlassian.net/browse/SV-102

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Add registration `/signup` page 
Remove unused `assets/images` directory

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/pages/signup.vue [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        <UiFormLabel>{{ $t('form.firstNameLabel') }}</UiFormLabel>
+        <SfInput name="firstName" autocomplete="given-name" v-model="firstNameModel" required />
+      </label>
+      <label>
+        <UiFormLabel>{{ $t('form.lastNameLabel') }}</UiFormLabel>
+        <SfInput name="lastName" autocomplete="family-name" v-model="lastNameModel" required />
+      </label>
+      <label>
+        <UiFormLabel>{{ $t('form.emailLabel') }}</UiFormLabel>
+        <SfInput name="email" autocomplete="off" v-model="emailModel" required />
```
Comment (justyna-13) [inline apps/web/pages/signup.vue]: type="email" to validate input

File: apps/web/pages/signup.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+      <p class="text-sm text-neutral-500 mt-0.5 mb-2">{{ $t('form.asterixHint') }}</p>
+
+      <SfButton type="submit" size="lg" class="w-full">
+        {{ $t('auth.signup.createButton') }}
+      </SfButton>
+    </form>
+
+    <UiOverlay :visible="isOpen">
+      <SfModal
```
Comment (justyna-13) [inline apps/web/pages/signup.vue:80]: modal on mobile should have 16px space on both sides 
Screenshot 2023-08-07 at 12 15 54

File: apps/web/pages/signup.vue [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      >
+        <header class="flex items-center flex-col">
+          <NuxtImg
+            src="/images/signup-success.svg"
+            :alt="$t('auth.signup.modal.imageAlt')"
+            width="192"
+            height="192"
+            class="my-6"
+          />
+          <h2 id="signUpModealTitle" class="mt-6 mb-4 font-bold typography-headline-4 md:typography-headline-3">
```
Comment (justyna-13) [inline apps/web/pages/signup.vue:97]: modal headline has font size 24px on both mobile and desktop

File: apps/web/pages/signup.vue [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    </form>
+
+    <UiOverlay :visible="isOpen">
+      <SfModal
+        v-model="isOpen"
+        class="max-w-[480px]"
+        tag="section"
+        role="alertdialog"
+        disable-click-away
+        aria-labelledby="signUpModealTitle"
```
Comment (justyna-13) [inline apps/web/pages/signup.vue:86]: signUpModealTitle => signUpModalTitle

File: apps/web/pages/signup.vue (hunk 3 above)
Comment (justyna-13) [inline apps/web/pages/signup.vue:97]: signUpModealTitle = > signUpModalTitle

File: apps/web/pages/signup.vue [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+          />
+          <h2 id="signUpModealTitle" class="mt-6 mb-4 font-bold typography-headline-4 md:typography-headline-3">
+            {{ $t('auth.signup.modal.heading') }}
+          </h2>
+        </header>
+        <UiAlert class="w-full p-4 mb-6 !justify-start typography-text-base" variant="neutral">
+          <i18n-t keypath="auth.signup.modal.description" tag="p" id="signUpModalDesc">
+            <template #information>
+              <SfLink
+                href="#"
```
Comment (justyna-13) [inline apps/web/pages/signup.vue:105]: “add information” link redirects to “My account” page

**Review by justyna-13 — COMMENTED** [review]
Great job! Some really small changes

File: apps/web/pages/signup.vue [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      </i18n-t>
+    </UiAlert>
+
+    <form
+      data-testid="signup-form"
+      class="flex flex-col md:border md:border-neutral-200 rounded-md gap-4 md:p-6"
+      @submit.prevent="open"
+    >
+      <label>
+        <UiFormLabel>{{ $t('form.firstNameLabel') }}</UiFormLabel>
```
[the remark this answers is not in the document]
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue:23]: Yep, thanks for reminder, today I established it with Jacek, before there was no * https://vsf.atlassian.net/browse/SV-148

File: apps/web/pages/signup.vue (hunk 1 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: yep it was done today around 2hours ago https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/files#diff-0fb18173332eefea105f2be197a7939105cd4ab3b6cd0620af508abe63e8d6feR32

File: apps/web/pages/signup.vue (hunk 2 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue:80]: I know but unfortunately this is not doable easy, but I'll do second approach

File: apps/web/pages/signup.vue (hunk 5 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue (hunk 3 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue (hunk 4 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+          <UiFormLabel>{{ $t('form.passwordLabel') }}</UiFormLabel>
+          <UiFormPasswordInput name="password" autocomplete="current-password" v-model="passwordModel" required />
+        </label>
+        <p class="text-xs text-neutral-500 mt-0.5 mb-2">{{ $t('form.passwordHint') }}</p>
+      </div>
+
+      <div class="flex items-center">
+        <SfCheckbox id="terms" v-model="termsAndConditionsModel" value="value" class="peer" required />
+        <label
+          class="ml-3 text-base text-gray-900 cursor-pointer font-body peer-disabled:text-disabled-900"
```
[the remark this answers is not in the document]
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+              </SfLink>
+            </template>
+          </i18n-t>
+        </label>
+      </div>
+
+      <div class="flex mb-2">
+        <SfCheckbox id="subscription" v-model="subscriptionsModel" value="value" class="peer mt-0.5" />
+        <label
+          class="ml-3 text-base text-gray-900 cursor-pointer font-body peer-disabled:text-disabled-900"
```
[the remark this answers is not in the document]
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue (hunk 6 above)
[the remark this answers is not in the document]
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/3ef4ca1e46b5919f61c260e77b442c12e6eb65c6

File: apps/web/pages/signup.vue (hunk 3 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue]: great catch https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/aec0989c6b2de9618fbaaa46817a305e7dcc88ca

File: apps/web/pages/signup.vue (hunk 2 above)
Comment (Szymon-dziewonski) [reply apps/web/pages/signup.vue:80]: actually it wasnt hard, my brain just froze :D https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/97/commits/922e7a4f55e9ab97dcb107aabfef59d696289dc6 thanks for checking this out

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**